src/main/java/no/nav/brregstub/mapper/RolleoversiktMapper.java

```diff
@@ -69,6 +85,33 @@ private static ResponseHeader mapTilResponseHeader(RolleoversiktTo to) {
         return responseHeader;
     }
 
+    private static ResponseHeader mapTilResponseHeader(RsRolleoversikt rsRolleoversikt) {
+        var responseHeader = new ResponseHeader();
+        responseHeader.setProssessDato(localDateToXmlGregorianCalendar(LocalDate.now()));
+        responseHeader.setTjeneste(TJENESTE_NAVN);
+        responseHeader.setFodselsnr(rsRolleoversikt.getFnr());
+        responseHeader.setHovedStatus(rsRolleoversikt.getHovedstatus());
+        var underStatus = new UnderStatus();
+        if (rsRolleoversikt.getUnderstatuser() == null || rsRolleoversikt.getUnderstatuser().isEmpty()) {
+            var underStatusMelding = new UnderStatusMelding();
+            underStatusMelding.setKode(0);
+            underStatusMelding.setValue(understatusKoder.get(0));
+
+            underStatus.getUnderStatusMelding().add(underStatusMelding);
+        } else {
+            for (Integer understatus : rsRolleoversikt.getUnderstatuser()) {
```

Comment:
Har forstått det slik at best practice er å benytte forEach heller enn for løkke der det er mulig.
https://stackoverflow.com/questions/16635398/java-8-iterable-foreach-vs-foreach-loop